modin/engines/ray/generic/io.py

```diff
@@ -41,6 +41,6 @@ def func(df):
             df.to_sql(**kwargs)
             return pandas.DataFrame()
 
-        result = qc._modin_frame._fold_reduce(1, func)
+        result = qc._modin_frame._apply_full_axis(1, func)
```

Comment:
Look at the `func` above that we pass here to `_fold_reduce`, it returns empty DataFrame, so the shape partitions changing (including indices and columns). `_fold_reduce` can't handle changing shape on both axis, when `_apply_full_axis` can.

Reply:
Okay, I see. Since we know that the result will be consist of empty partitions, may be, make sense pass to `_apply_full_axis` some values `new_index` and `new_columns` which are applied for empty dataframe?

Reply:
done